### DISCUSSION TITLE
Add `filament:about` artisan command

### DISCUSSION THREAD

**Discussion by dissto** [filamentphp/filament#14578] [Ideas]
This currently only exists as a proof-of-concept. But before I spend more time on this, I would like to know if you think this is worthy to PR to the Filament core? 

A command that outputs details about the current Filament installation. Something like this:

![Screenshot 2024-10-09 205351](https://github.com/user-attachments/assets/75b6c44e-a136-4fb9-962f-ce8867bd1800)

Or additionally more details about a given panel. For example: 

![Screenshot 2024-10-09 234619](https://github.com/user-attachments/assets/d0ca7ef9-c44c-4768-8d65-5a793ab81c37)

Thinking a bit further. This could introduce the ability for plugin developers to show custom information when detailing a single plugin. For example: 

`php artisan filament:about --plugin="awcodes/filament-table-repeater"` 

![Screenshot 2024-10-20 115216](https://github.com/user-attachments/assets/6ff3f327-d2a9-489e-a88c-e9b2f1ecf7ea)

I thought about making this a plugin. But I think this would immediately loose all its value, when there are additional steps required to get this information (aka installing another plugin) 🤔 

Just an idea. What do you think @danharrin?

**Comment by danharrin** [comment]
Hey, can you elaborate a bit about when this would be used instead of `php artisan about`?

**Reply by dissto** [reply]
Well basically to get all the relevant data for your Filament installion. I know you could filter with `--only=filament`. Perhaps the example above is a little _over the top_ but I think having a command under the `filament` namespace would be good even if its only the addition of the `Optimization` section and perhaps a list of panels? 🤔

**Reply by danharrin** [reply]
I think the Optimization section would be good to add to our existing `about` details, and then you can add a `filament:about` alias for `php artisan about --only=filament` if you want :)

**Reply by dissto** [reply]
I PR'ed...maybe it could be worth it to follow up with a plugin at some point to have a more detailed/adavanced output for that command. Ill think about it 🤪

**Reply by leandrocfe** [reply]
I remember doing something like that..
https://github.com/filamentphp/filament/pull/13850
